README.md

```diff
@@ -1 +1,915 @@
+
+```text
+$ rasactl help model
+manage models for Rasa X / Enterprise
+
+Usage:
+  rasactl model [command]
+
+Available Commands:
+  delete      delete a model from Rasa X / Enterprise
+  download    download a model from Rasa X / Enterprise
+  list        list models stored in Rasa X / Enterprise
+  tag         tag a model in Rasa X / Enterprise
+  upload      upload model to Rasa X / Enterprise
+```
+
+### The `model delete` command
+
+Delete a model from Rasa X / Enterprise.
+
+```text
+Usage:
+  rasactl model delete [DEPLOYMENT-NAME] MODEL-NAME [flags]
+
+Aliases:
+  delete, del
+```
+
+```text
+Examples:
+  # Delete the 'example-model' model (use the currently active deployment).
+  $ rasactl model delete example-model
+
+  # Delete the 'example-model' model for the 'my-deployment' deployment.
+  $ rasactl model delete my-deployment example-model
+```
+
+```text
+Flags:
+  -h, --help   help for delete
+```
+
+### The `model download` command
+
+Download a model from Rasa X / Enterprise to your local machine.
+
+```text
+Usage:
+  rasactl model download [DEPLOYMENT-NAME] MODEL-NAME [DESTINATION] [flags]
+```
+
+```text
+Examples:
+  # Download the 'example-model' model (use the currently active deployment).
+  # If the destination is not defined, the model will be stored in a current working directory.
+  $ rasactl model download example-model
+
+  # Download the 'example-model' model for the 'my-deployment' deployment
+  # and store it in the /tmp directory.
+  $ rasactl model download my-deployment example-model /tmp/example-model.tar.gz
+```
+
+```text
+Flags:
+  -h, --help   help for download
+```
+
+### The `model list` command
+
+List all models stored in Rasa X / Enterprise.
+
+```text
+Usage:
+  rasactl model list [DEPLOYMENT-NAME] [flags]
+
+Aliases:
+  list, ls
+```
+
+```text
+Examples:
+  # List all models (use the currently active deployment).
+  $ rasactl model list
+
+  # List all models for the 'my-deployment' deployment.
+  $ rasactl model list my-deployment
+```
+
+```text
+Flags:
+  -h, --help   help for list
+```
+
+### The `model tag` command
+
+Create a tag and assign it to a given model.
+
+Rasa Enterprise allows multiple versions of an assistant to be run simultaneously and served to different users. By default, two environments are defined:
+
+- production
+- worker
+
+ If you want to activate a model you have to tag it as 'production'.
+
+ Learn more [here](https://rasa.com/docs/rasa-x/enterprise/deployment-environments/).
+
+```text
+Usage:
+  rasactl model tag [DEPLOYMENT-NAME] MODEL-NAME TAG [flags]
+```
+
+```text
+Examples:
+  # Tag the 'my-model' model as 'production' (use the currently active deployment)
+  $ rasactl model tag my-model production
+
+  # Tag the 'my-model' with the 'test' tag within the 'my-deployment' deployment.
+  $ rasactl model tag my-deployment my-model test
+```
+
+```text
+Flags:
+  -h, --help   help for tag
+```
+
+### The `model upload` command
```

Comment:
This should be up with the other model commands

Reply:
hmm, it is, no?

Reply:
ah my bad, i mixed it up with the `### Upload a model to Rasa X` usage command!